src/docs/sphinx/developerGuide/Contributing/Doxygen.rst

```diff
@@ -239,4 +239,4 @@ Example
 Current Doxygen
 ====================================
 
-`Link to Doxygen <../../../doxygen_output/html/classes.html>`__
+`Link to Doxygen <../../../../doxygen_output/html/classes.html>`__
```

Comment:
You can also directly link to the specific class or main entry point for API.

Reply:
I added direct links to class API in various places

Reply:
I see I triggered a clean-up wave in the Doxygen pages. Nice!
This is already much better.

Reply:
Documentation is always a work in progress. Thanks for the suggestions!